cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/Joiner.java

```diff
@@ -36,8 +39,21 @@
    * @return returns join key
    * @throws Exception if there is some error getting the join key
```

Comment:
include a `@deprecated` in the javadoc and mention that getJoinKeys() should be used instead.

Reply:
done